include/aca_comm_mgr.h

```diff
@@ -16,7 +16,7 @@
 #define ACA_COMM_MGR_H
 
 // #include "aca_net_state_handler.h"
-#include "cppkafka/buffer.h"
+// #include "cppkafka/buffer.h"
```

Comment:
you can go ahead to remove these two commented lines. No need to keep them around :)